<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShutDownPCClass</class>
 <widget class="QMainWindow" name="ShutDownPCClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShutDownPC</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <pointsize>100</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Turn off IN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Turn off AFTER</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="btnLayout">
        <item>
         <widget class="QTimeEdit" name="timeEdit">
          <property name="font">
           <font>
            <pointsize>100</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::HourSection</enum>
          </property>
          <property name="timeSpec">
           <enum>Qt::LocalTime</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="timerLayout">
        <item>
         <widget class="QPushButton" name="startBtn">
          <property name="font">
           <font>
            <pointsize>100</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>START</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeBtn">
          <property name="font">
           <font>
            <pointsize>100</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>CLOSE</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ShutDownPC.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>ShutDownPCClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
